src/core/core.controller.js

```diff
@@ -365,9 +365,11 @@ module.exports = function(Chart) {
 			me.updateLayout();
 
 			// Can only reset the new controllers after the scales have been updated
-			helpers.each(newControllers, function(controller) {
-				controller.reset();
-			});
+			if (me.options.animation.duration) {
```

Comment:
Animations are also disabled when `animation` is false (or null)

```javascript
if (me.options.animation && me.options.animation.duration) {
    // ...
}
```

Reply:
thanks! done